cmd/server.go

```diff
@@ -228,15 +233,15 @@ func (o *serverOption) runE(cmd *cobra.Command, args []string) (err error) {
 			reflection.Register(gRPCServer)
 		}
 		server.RegisterRunnerServer(s, remoteServer)
-		log.Printf("gRPC server listening at %v", lis.Addr())
+		serverLogger.Info("gRPC server listening at ", lis.Addr())
```

Comment:
The correct format is like blew:

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/a46a4782-4c24-4a2d-ab32-a7936722940a)

```
func (l Logger) Info(msg string, keysAndValues ...any) {
```

The first parameter is the message, and others should be pair.

Reply:
yep, you're right, I'll revise it again.